.vale.ini

```diff
@@ -0,0 +1,31 @@
+# Config vale. More information at https://docs.errata.ai/vale/config
+StylesPath = .github/styles
+MinAlertLevel = suggestion
+
+Packages = Google
+
+[*.md]
+# Ignore code injection which start with {{...
+BlockIgnores = {{.*
+
+# Custom syle
+# BasedOnStyles = Blog
+
+Blog.ComposedWords = YES
+Blog.NamingConventions = YES
+Blog.Typos = YES
+
+# Google:
+Google.FirstPerson = YES # Avoid first-person pronouns such as I, me, ...'.
+Google.GenderBias = YES # Avoid gendered profession
+Google.OxfordComma = YES
+Google.Quotes = YES # Commas and periods go inside quotation marks.
+Google.Spelling = YES #In general, use American spelling (word ending with 'nised' or 'logue')
+Google.We = YES # Try to avoid using first-person plural
+
+# Those rules are not repected a lot
+# Google.Passive = YES # In general, use active voice instead of passive voice.
+# Google.Will = YES # Avoid using will
```

Comment:
Personally I'd love to see these implemented. And they should be respected! 😁 (I suppose I could let "will" slide, but I do prefer programmatic text to stay in present tense: "Applying x prop changes the y to z" instead of "Applying x prop will change the y to z.")

Reply:
```suggestion
# Google.Passive = YES # In general, use active voice instead of passive voice.
Google.Will = YES # Avoid using will
```

Reply:
I'm doing the setup, but feel free to modify the config. I never write docs on core, so I might never be impacted by those config files ;)